catalog-info.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: kost
+  title: kost
+  annotations:
+    backstage.io/techdocs-ref: dir:.
```

Comment:
We don't need this since this project doesn't docs :)

Reply:
I don't think it's a big deal :shrug: 

Without the annotation, backstage will show a "clean" error indicating that the docs aren't configured. With it, it will probably display an error on the docs page (since there are no docs)